UI/GovernmentWnd.cpp

```diff
@@ -673,8 +671,8 @@ void GovernmentWnd::PolicyPalette::CompleteConstruction() {
                                                  GG::FORMAT_CENTER, std::make_shared<CUILabelButtonRepresenter>());
         AttachChild(button);
         button->CheckedSignal.connect(
-            boost::bind(&GovernmentWnd::PolicyPalette::ToggleAvailability, this, Availability(n)));
-        button->SetCheck(m_availabilities_state.GetAvailability(Availability(n)));
```

Comment:
I think the `static_cast` does the same as the constructor-like `Availability(n)` anyway. Probably works out to be the same code generation...

Reply:
In that case I think it's better to remove it.